src/templates/schedule.html

```diff
@@ -10,6 +10,7 @@
   <ul>
     <li>Room 1 Lynnhaven <a href="http://dominionenterprises.adobeconnect.com/devcon1">http://dominionenterprises.adobeconnect.com/devcon1</a></li>
     <li>Room 2 York <a href="http://dominionenterprises.adobeconnect.com/devcon2">http://dominionenterprises.adobeconnect.com/devcon2</a></li>
+    <li>Room 3 Rappahannock <a href="http://dominionenterprises.adobeconnect.com/devcon3">http://dominionenterprises.adobeconnect.com/devcon3</a></li>
```

Comment:
Yeah, i should. We're doing an "un-con" this year in that room so it won't be available remotely. I should add a note about that.